backend/src/libs/sequelize.js

```diff
@@ -1,7 +1,7 @@
 const Sequelize = require('sequelize');
 const log         = require('./log')(module);
 
-const sequelize = new Sequelize('postgres://vagrant:vagrant@127.0.0.1:5432/pkeeper');
+const sequelize = new Sequelize('postgres://vagrant:vagrant@postgres:5432/pkeeper');
```

Comment:
Then you might as well rename db user :) it is kinda weird to have user named vagrant while interacting from docker